packages/react-composites/tests/browser/chat/FileSharing.test.ts

```diff
@@ -194,3 +194,39 @@ test.describe('Filesharing SendBox Errorbar', async () => {
     expect(await page.screenshot()).toMatchSnapshot('filesharing-sendbox-file-upload-in-progress-error.png');
   });
 });
+
+test.describe('Filesharing Message Thread', async () => {
+  test.skip(isTestProfileStableFlavor());
+
+  test.beforeEach(async ({ pages, users, serverUrl }) => {
+    await chatTestSetup({ pages, users, serverUrl });
+  });
+
+  test('contains File Download Card', async ({ serverUrl, users, pages }) => {
+    const testMessageText = 'Hello!';
+    const page0 = pages[0];
+    await page0.goto(
+      buildUrl(serverUrl, users[0], {
+        useFileSharing: 'true',
+        uploadedFiles: JSON.stringify([
+          {
+            name: 'SampleFile1.pdf',
+            extension: 'pdf',
+            url: 'https://sample.com/SampleFile.pdf',
+            uploadComplete: true
+          }
+        ])
+      })
+    );
+    // console.log(`xkcd: ${buildUrl(serverUrl, users[1])}`);
+    await waitForChatCompositeToLoad(page0);
+    await sendMessage(page0, testMessageText);
+    await waitForMessageDelivered(page0);
```

Comment:
Looks like it's not waiting for the message to have been delivered:
![image](https://user-images.githubusercontent.com/2684369/162528298-7198a27b-1b89-42bb-8edf-5fba90b27508.png)

Reply:
`waitForMessageDelivered` looks poorly implemented and waits for _any_ messages to have been delivered